src/main/java/com/jcabi/github/RtReleases.java

```diff
@@ -64,6 +65,16 @@
     private final transient Request request;
 
     /**
+     * RESTful API releases request.
+     */
+    private final transient Request latest;
+
+    /**
+     * RESTful API releases request.
+     */
+    private final transient RequestURI tagged;
```

Comment:
@weisebrazil and you also don't need this field, it can also be created during `tagged()` method call